sim/rogue/talents.go

```diff
@@ -11,101 +10,62 @@ import (
 
 func (rogue *Rogue) ApplyTalents() {
 	rogue.applyMurder()
-	rogue.applySlaughterFromTheShadows()
 	rogue.applySealFate()
 	rogue.applyWeaponSpecializations()
-	rogue.applyCombatPotency()
-	rogue.applyFocusedAttacks()
+	rogue.applyWeaponExpertise()
 	rogue.applyInitiative()
 
-	rogue.AddStat(stats.Dodge, core.DodgeRatingPerDodgeChance*2*float64(rogue.Talents.LightningReflexes))
-	rogue.PseudoStats.MeleeSpeedMultiplier *= []float64{1, 1.03, 1.06, 1.10}[rogue.Talents.LightningReflexes]
-	rogue.AddStat(stats.Parry, core.ParryRatingPerParryChance*2*float64(rogue.Talents.Deflection))
+	rogue.AddStat(stats.Dodge, core.DodgeRatingPerDodgeChance*1*float64(rogue.Talents.LightningReflexes))
+	rogue.AddStat(stats.Parry, core.ParryRatingPerParryChance*1*float64(rogue.Talents.Deflection))
 	rogue.AddStat(stats.MeleeCrit, core.CritRatingPerCritChance*1*float64(rogue.Talents.Malice))
 	rogue.AddStat(stats.MeleeHit, core.MeleeHitRatingPerHitChance*1*float64(rogue.Talents.Precision))
 	rogue.AddStat(stats.SpellHit, core.SpellHitRatingPerHitChance*1*float64(rogue.Talents.Precision))
-	rogue.AddStat(stats.Expertise, core.ExpertisePerQuarterPercentReduction*5*float64(rogue.Talents.WeaponExpertise))
-	rogue.AddStat(stats.ArmorPenetration, core.ArmorPenPerPercentArmor*3*float64(rogue.Talents.SerratedBlades))
+	// Serrated Blades armor pen here
```

Comment:
Is this still TODO?

Reply:
Yes, in game testing is required to get the proper amount of armor reduction per rank based on level (25/40/50/60).

Reply:
Updated for now. May need to update later for accuracy